com.cuperino.qprompt.json

```diff
@@ -23,14 +23,26 @@
    ],
    "modules": [
       {
+         "name": "qhotkey",
+         "buildsystem": "cmake-ninja",
+         "sources": [
+            {
+               "type": "git",
+               "url": "https://github.com/Skycoder42/QHotkey.git",
+               "branch": "master",
```

Comment:
you don't want a branch here it will break the build at the next commit.

Reply:
I see. Thanks for the tip and fix.